charmcraft/templates/init-simple/tox.ini.j2

```diff
@@ -2,27 +2,27 @@
 # See LICENSE file for licensing details.
 
 [tox]
-skipsdist=True
+no_package = True
 skip_missing_interpreters = True
-envlist = fmt, lint, unit
+env_list = format, lint, unit
 
 [vars]
-src_path = {toxinidir}/src/
-tst_path = {toxinidir}/tests/
-;lib_path = {toxinidir}/lib/charms/operator_name_with_underscores
-all_path = {[vars]src_path} {[vars]tst_path} 
+src_path = {tox_root}/src
+tests_path = {tox_root}/tests
+;lib_path = {tox_root}/lib/charms/operator_name_with_underscores
```

Comment:
Is this intentionally commented out? If so it should probably be noted to uncomment it down in the lint testenv